{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/utils/emails.py

```diff
@@ -5,7 +5,14 @@
 from django.conf import settings
 from django.core.mail.message import EmailMultiAlternatives
 from django.template.loader import render_to_string
-from premailer import transform
+from premailer import transform as inline_css
+
+
+def get_html_body(template, context):
+    # two steps:
+    # Render the django jinja template to string with passed context
+    # Use premailer to inline CSS styles for weird email clients
+    return inline_css(render_to_string(template, context))
```

Comment:
Refactored this for two reasons:

- Easier to write tests
- `transform()` is vague
- `render_to_string()` is vague